Things_You_Should_Know/kind_property.md

```diff
@@ -16,13 +16,15 @@ You will need to manually set the **kind property** if you are creating resource
 
 The following table contains a list of valid values for Kind property:
 
-|Kind                | Resource Type                  |
-|--------------------|--------------------------------|
-|`app`               | Web app                        |
-|`api`               | API app                        |
-|`functionapp`       | Function App                   |
-|`functionapp,linux` | Linux Consumption Function app |
-|`hyperV`            | Windows Container Web App      |
+|Kind                      | Resource Type                  |
+|--------------------------|--------------------------------|
+|`app`                     | Windows Web app                |
+|`app,linux`               | Linux Web app                  |
+|`hyperV`                  | Windows Container Web App      |
+|`app,container,windows`   | Windows Container Web App      |
+|`app,linux,container`     | Linux Container Web App        |
+|`functionapp`             | Function App                   |
+|`functionapp,linux`       | Linux Consumption Function app |
```

Comment:
You missed a couple less known ones I think.

functionapp,linux,container,kubernetes - Function container app on ARC
functionapp,linux,kubernetes - Function code app on ARC

@andimarc does that look right?  Not sure why I don't see the equivalent ones for regular web apps defined in our assets files.

Reply:
added those to the list in the recent commit

Reply:
There's still one missing for ARC: 'app,linux,container,kubernetes'.

Aside from that, it looks right.

Reply:
thanks @andimarc!